docs/getting_started/installation.md

```diff
@@ -206,6 +201,22 @@ Only available when running with Docker or `mix release`.
 Defaults to `9000` when running with Docker.<br/>
 Only available when running with Docker or `mix release`.
 
+###### Distribution NODES_LIST specific:
+* `JF_DIST_NODES` - space-separated list of other Jellyfishes to connect to.<br/>
+Defaults to `""`.<br/>
+Example: `JF_DIST_NODES="jellyfish1@127.0.0.1 jellyfish2@127.0.0.1"`.<br/>
+This list can also include ourselves so that you can pass the same value
+to every Jellyfish.
+Note: Jellyfish connection to other Jellyfish is transitive meaning that
+when you connect to one Jellyfish you also connect to all other Jellyfishes
+this one was connected to.
+
+###### Distribution DNS specific:
+* `JF_DIST_QUERY` - query which is sent do DNS. <br/>
+Returned list of IPs from DNS is used for creating distribution node name in the format `<NODE_BASENAME>@<IP_ADDRESS>`.  
```

Comment:
I belive we should no longer refer to `NODE_BASENAME` as we removed this env. We need to also mention (probably in JF_DIST_NODE_NAME?) that in the case of DNS distribution, every JF has to have the same `nodename` part of `nodename@hostname`.

Reply:
I mentioned that [here](https://github.com/jellyfish-dev/jellyfish-docs/blob/06dcde196f461be3953dfd4efb10c5b91c94cfc2/docs/cluster.md?plain=1#L79-L80) but if you want I can also mention it with `JF_DIST_NODE_NAME`.

Reply:
Please do